metagraph/src/annotation/binary_matrix/row_diff/row_diff.hpp

```diff
@@ -148,6 +152,70 @@ BinaryMatrix::SetBitPositions RowDiff<BaseMatrix>::get_row(Row row) const {
     return result;
 }
 
+template <class BaseMatrix>
+std::vector<BinaryMatrix::SetBitPositions>
+RowDiff<BaseMatrix>::get_rows(const std::vector<Row> &row_ids) const {
+    // diff rows annotating nodes along the row-diff paths
+    std::vector<Row> rd_ids;
+    rd_ids.reserve(row_ids.size() * 2);
```

Comment:
Let's make this a compile-time constant for now.

Reply:
You mean `2`?

I think this is a super minor thing. For large batches, this should be negligible. Maybe we should just remove these reserves?

Reply:
It's hard to predict a value for this since the graph topology is so variable. All we know really is that we can bound it above by the max row diff path length.

Reply:
Yeah, but that could add an overhead of the worst case. I will just remove these reserves. They are insignificant anyway. Most of the complexity is in traversing and querying diff rows.

Reply:
Never mind, why not to reserve, we know for sure it's not going to be smaller than `row_ids.size()`.

I'll add a global constant.